src/DssSpell.sol

```diff
@@ -19,28 +19,32 @@ pragma solidity 0.8.16;
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
 
-import { DssInstance, MCD } from "dss-test/MCD.sol";
-import { D3MInit, D3MCommonConfig, D3M4626PoolConfig, D3MOperatorPlanConfig } from "src/dependencies/dss-direct-deposit/D3MInit.sol";
-import { D3MInstance } from "src/dependencies/dss-direct-deposit/D3MInstance.sol";
-
 interface ProxyLike {
     function exec(address target, bytes calldata args) external payable returns (bytes memory out);
 }
```

Comment:
Proxy interface is also present in dss-interfaces (src/dapp/DSPauseProxyAbstract.sol)

Reply:
Noticed on all previous spells it is declared locally instead of importing from dss-interfaces. Does this classifies as an "archive pattern"?

Reply:
I don't have strong opinion on this.
Though AFAIK, we use static interface when the interface only has a very few functions like this case as a convention.

Reply:
`dss-interfaces` hasn't been updated in a long time. We've been preferring to use inline interfaces instead.